packages/sources/tradingeconomics-test/src/config/index.ts

```diff
@@ -0,0 +1,31 @@
+export const defaultEndpoint = 'price'
+export const customSettings = {
+  API_ENDPOINT: {
+    description: 'The HTTP URL to retrieve data from',
+    type: 'string',
+    default: 'https://api.tradingeconomics.com/markets',
+    required: true,
+  },
+  WS_API_ENDPOINT: {
+    description: 'The WS URL to retrieve data from',
+    type: 'string',
+    default: 'wss://stream.tradingeconomics.com/',
+    required: true,
+  },
```

Comment:
I don't think we need to mark these as `required: true`. Since they have defaults, they'll always have a value.